src/components/Rating/Rating.tsx

```diff
@@ -0,0 +1,330 @@
+import React, { Component } from 'react'
+import CSS from "csstype";
+
+export interface propTypes {
+  className?: string;
+  edit?: boolean;
+  half?: boolean;
+  value: number;
+  count: number;
+  char?: string;
+  size: number;
+  color1?: string;
+  color2?: string;
+  onChange: any;
```

Comment:
onChange is a function. Because it works as a callback function, my suggestion is to change the type of  `any` to `() => void`